Rakefile

```diff
@@ -111,7 +111,7 @@ namespace :build do
       env_vars = ''
       env_vars += 'MOOC_LOCALIZE=1 ' if CDO.localize_apps
       env_vars += 'MOOC_DIGEST=1 ' unless rack_env?(:development)
-      RakeUtils.system "#{env_vars} grunt"
+      RakeUtils.system "#{env_vars} grunt build"
```

Comment:
Historically, I believe we had issues where build didn't always update CSS properly. I don't know if that's still an issue or not - I think someone maybe investigated recently and couldn't repro any problems.

tbh, i'm just going to continue using `grunt` commands directly regardless, because that's what has always worked for me and that's what I know.